mediaplatform_jwp/jwplatformapi/managementapi.py

```diff
@@ -2,10 +2,10 @@
 Interactions with the JWP management API.
```

Comment:
suggest calling this module management.py given api is in the parent directory

Reply:
yep it makes sense, I wanted to do it but I forgot. Initially it was called management.py but it was conflicting with the management folder/module inside mediaplatform_jwp so I changed it to managementapi.py. Now is one level down so it won't conflict.